server/status_test.go

```diff
@@ -46,18 +46,37 @@ func startStatusServer() *httptest.Server {
 	return httpServer
 }
 
-// TestStatusLocalStacks verifies that goroutine stack traces are available
-// via the /_status/local/stacks endpoint.
-func TestStatusLocalStacks(t *testing.T) {
+// TestStatus verifies that status endpoints return expected results.
+func TestStatus(t *testing.T) {
 	s := startStatusServer()
 	defer s.Close()
-	body, err := getText(s.URL + statusLocalStacksKey)
-	if err != nil {
-		t.Fatal(err)
+
+	testCases := []struct {
+		keyPrefix string
+		expected  string
+	}{
+		{statusKeyPrefix, "{}"},
```

Comment:
I'm surprised the expected output doesn't have to change for each content-type. Am I missing something here?